augur/__main__.py

```diff
@@ -2,11 +2,16 @@
 Stub function and module used as a setuptools entry point.
 """
 
+import sys
 import augur
 from sys import argv, exit
 
 # Entry point for setuptools-installed script and bin/augur dev wrapper.
 def main():
+    # Support non-Unicode encodings by replacing Unicode characters instead of erroring.
+    sys.stdout.reconfigure(errors="backslashreplace")
+    sys.stderr.reconfigure(errors="backslashreplace")
```

Comment:
_non-blocking_

We might also want to set `newlines = None` to ensure universal newlines is enabled?

Reply:
Done: fd8516816d95fea9e5c08a901a358b0a024f5521